tests/test_kmeans_segmentation.py

```diff
@@ -0,0 +1,16 @@
+import unittest
+import numpy as np
+from Segmentation.kmeans_segmentation import segment_blue_nuclei
+from data_types import SegmentedImage
+
+
+class TestShapeFeatures(unittest.TestCase):
```

Comment:
The same name as in the _test_shape_features_file.py_?

Reply:
This is just a mistake